### PR TITLE
On virtual product, the delivery time shouldn't display

### PR DESCRIPTION
**Pull request by PeeyushAgrawal**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop 
| Description?  | Fixes the issue #17542
| Type?         | bug fix 
| Category?     | FO
| BC breaks?    | no
| Deprecations? |o
| Fixed ticket? | #17542.
| How to test?  |Steps to reproduce the behavior:<br>1. Go to 'Products setting' in shop parameters tab on the BO and set "Delivery time of in-stock products".<br>2. Now go to the catalog page and create a new virtual product. <br>3. Click on preview - here you will find that the delivery time is getting display. 

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/17569)

### CODE REVIEW

**Review by Progi1984 — DISMISSED** [review]

File: themes/classic/templates/catalog/_partials/product-prices.tpl [hunk 1]
```diff
@@ -97,6 +97,7 @@
       {/if}
       {hook h='displayProductPriceBlock' product=$product type="price"}
       {hook h='displayProductPriceBlock' product=$product type="after_price"}
+      {if $product.is_virtual	== 0}
       {if $product.additional_delivery_times == 1}
```
Comment (PierreRambaud) [inline themes/classic/templates/catalog/_partials/product-prices.tpl]: Must be indented otherwise it's confusing

**Review by PierreRambaud — CHANGES_REQUESTED** [review]
See my comments

File: themes/classic/templates/catalog/_partials/product-prices.tpl (hunk 1 above)
Comment (PeeyushAgrawal) [reply themes/classic/templates/catalog/_partials/product-prices.tpl]: changes indented.

**Review by Progi1984 — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by SD1982** [comment]
This pull request correctly fix this issue : [#17542 ](https://github.com/PrestaShop/PrestaShop/issues/17542). Thanks @PeeyushAgrawalWebkul

**Comment by PierreRambaud** [comment]
Thanks @PeeyushAgrawalWebkul

**Pull request merged**